velox/vector/BaseVector.h

```diff
@@ -51,6 +51,11 @@ class VectorPool;
 class BaseVector;
 using VectorPtr = std::shared_ptr<BaseVector>;
 
+using column_index_t = uint32_t;
```

Comment:
This sounds like something at type level, can we move it into `Type.h`?

Reply:
Thanks @Yuhta. That works as well.